go-controller/pkg/node/gateway_shared_intf_linux.go

```diff
@@ -81,14 +81,14 @@ func setupLocalNodeAccessBridge(nodeName string, subnets []*net.IPNet) error {
 }
 
 func addSharedGatewayIptRules(service *kapi.Service, nodeIP *net.IPNet) {
-	rules := getGatewayIPTRules(service, service.Spec.ClusterIP, nodeIP)
+	rules := getGatewayIPTRules(service, "", nodeIP)
```

Comment:
Why is this necessary though? The condition in `getGatewayIPTRules` will force the usage of `service.Spec.ClusterIP` anyways.